target/linux/ipq807x/base-files/etc/board.d/02_network

```diff
@@ -30,6 +30,9 @@ ipq807x_setup_interfaces()
 	xiaomi,ax9000)
 		ucidef_set_interfaces_lan_wan "eth0 eth1 eth2 eth3" "eth4"
 		;;
+	zyxel,nbg7815)
```

Comment:
Doesn't Qnap use the same network config?

Reply:
no '"ucidef_set_interfaces_lan_wan "eth0 eth1 eth2 eth3 eth4" "eth5"'

Reply:
Yes, looks similar.
Original Zyxel NBG7815 configuration is here:
https://github.com/kirdesde/nbg7815_gpl/blob/e816d72c67037186f2d5c10639f609649cbcce10/qca/src/linux-4.4/arch/arm64/boot/dts/qcom/qcom-ipq807x-nbg7815.dts

Reply:
Not just similar, but unless I am blind its exactly the same.
So, please confirm that and if so, there is no need for duplication.